Formula/csound.rb

```diff
@@ -12,6 +12,7 @@ class Csound < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "eigen" => :build
```

Comment:
Why is it a build-only dependency? Does it link statically?

Reply:
Eigen is a C++ template library—all header files. There isn’t, say, a libeigen to link to.